peer.go

```diff
@@ -1099,7 +1099,9 @@ func (p *peer) PingTime() int64 {
 func (p *peer) queueMsg(msg lnwire.Message, doneChan chan struct{}) {
 	select {
 	case p.outgoingQueue <- outgoinMsg{msg, doneChan}:
+		peerLog.Infof("queued msg successfully")
```

Comment:
`Debugf`?

Reply:
I guess this will happen quite often, so maybe even `Trace` 😛

Reply:
Guessing this was left over as part of a debugging session? It should either be removed, or be demoted to a `Tracef` logging level.

Reply:
Ignore the second commit, was using the latter commit to try and reproduce bugs on travis. Reverting to original PR